### PR TITLE
[Network] Update based on apiview comments

### PR DESCRIPTION
**Pull request by fengzhou-msft**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/network/Azure.ResourceManager.Network/src/autorest.md [hunk 1]
```diff
@@ -229,10 +230,22 @@ directive:
...
       $.NextHopParameters.properties.targetNicResourceId['x-ms-format'] = 'arm-id';
       $.NextHopResult.properties.routeTableId['x-ms-format'] = 'arm-id';
+      $.Direction['x-ms-enum'].name = 'TrafficDirection';
+      $.ConnectivityIssue.properties.origin['x-ms-enum'].name = 'ConnectivityIssueOrigin';
+      $.ConnectivityIssue.properties.severity['x-ms-enum'].name = 'ConnectivityIssueSeverity';
+      $.ConnectivityParameters.properties.protocol['x-ms-enum'].name = 'NetworkProtocol';
+      $.PacketCaptureStorageLocation.properties.storageId['x-ms-format'] = 'arm-id';
+      $.TroubleshootingProperties.properties.storageId['x-ms-format'] = 'arm-id';
+      $.FlowLogProperties.properties.storageId['x-ms-format'] = 'arm-id';
+      $.FlowLogPropertiesFormat.properties.storageId['x-ms-format'] = 'arm-id';
```
Comment (ArthurMa1978) [inline sdk/network/Azure.ResourceManager.Network/src/autorest.md:240]: $.ConnectivityIssue.properties.origin['x-ms-enum']['name'] = 'ConnectivityIssueOrigin';
      $.ConnectivityParameters.properties.protocol['x-ms-enum']['name'] = 'NetworkWatcherProtocol';

Comment (fengzhou-msft) [reply sdk/network/Azure.ResourceManager.Network/src/autorest.md:240]: The [description for protocol](https://github.com/Azure/azure-rest-api-specs/blob/889a0f6bf32c19174e190c4a5dcc4c74804732c6/specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/networkWatcher.json#L3282) is `Network protocol`, so maybe we can use `NetworkProtocol` although it's only used in networkWatcher.

Comment (ArthurMa1978) [reply sdk/network/Azure.ResourceManager.Network/src/autorest.md:240]: > The [description for protocol](https://github.com/Azure/azure-rest-api-specs/blob/889a0f6bf32c19174e190c4a5dcc4c74804732c6/specification/network/resource-manager/Microsoft.Network/stable/2021-02-01/networkWatcher.json#L3282) is `Network protocol`, so maybe we can use `NetworkProtocol` although it's only used in networkWatcher.

NetworkProtocal is not good for this RP as we have too many protocals

Comment (fengzhou-msft) [reply sdk/network/Azure.ResourceManager.Network/src/autorest.md:240]: I see.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**